strr-platform-web/app/components/form/platform/ContactInfo.vue

```diff
@@ -185,11 +176,19 @@ watch(secondaryRep, async () => {
             id-prefix="platform-secondary-rep"
             :prepopulate-name="false"
             name-divider
-            :error-name="secondaryRepNameErr"
-            :error-details="secondaryRepDetailsErr"
+            :error-name="hasFormErrors(secondaryRepFormRef, ['firstName', 'lastName'])"
+            :error-details="
+              hasFormErrors(secondaryRepFormRef, ['position', 'phone.countryCode', 'phone.number', 'email'])
+            "
           />
         </UForm>
       </ConnectPageSection>
     </div>
   </div>
 </template>
+<style>
+/* smooth scroll not working without this, add to layer? */
+html {
+  scroll-behavior: smooth;
```

Comment:
Did you try adding it to the routeRules? I saw that option there, but I've tried it

Reply:
I think this is only for hash links? I couldn't find anything for general app scroll behaviour https://nuxt.com/docs/guide/recipes/custom-routing#scroll-behavior-for-hash-links